CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+* Support for Boolean data-type is added to `dpctl.tensor.ceil`, `dpctl.tensor.floor`, and `dpctl.tensor.trunc` [gh-2033](https://github.com/IntelPython/dpctl/pull/2033)
```

Comment:
Do we need to update docstrings?
Currently it states:
> expected to have a real-valued data type

and according to the array spec it seems not including bool dtype.

Reply:
I think updating makes sense, for consistency.

My thought would be either
> boolean or real-valued data type
or
> a non-complex data type

I lean towards the former.

Reply:
My vote is also in favour of `boolean or real-valued data type`